monitor/services/src/test/java/org/datacleaner/monitor/configuration/ComponentStoreImplTest.java

```diff
@@ -47,7 +47,7 @@ private Repository getRepositoryMock() {
 
     private RepositoryFolder getRepositoryFolderMock() {
         RepositoryFolder repositoryFolder = createNiceMock(RepositoryFolder.class);
-        expect(repositoryFolder.getFolder(ComponentStoreImpl.FOLDER_NAME)).andReturn(getComponentsFolderMock()).anyTimes();
+        expect(repositoryFolder.getOrCreateFolder(ComponentStoreImpl.FOLDER_NAME)).andReturn(getComponentsFolderMock()).anyTimes();
```

Comment:
I think that really depends on the test case. Sometimes you want to ensure exact amount of invocations, and sometimes it may be too far away from the control of the thing you're testing. In the latter case you shouldn't require a specific number of invocations because the role of the mock is just to "behave like an implementation would".